README.md

```diff
@@ -37,6 +37,10 @@ Note: for help on installing extensions, consult the [extensions chapter of the
 
 Recalculate recipients is a simple extension with no configuration options. Once enabled it will start working automatically; recalculating recipients just before each mailing is sent. To stop recalculating recipients, simply disable the extension.
 
+## Help
+
+If you have any questions regarding this extension that are not answered in this README [or in the documentation], please check post a question on http://civicrm.stackexchange.com or contact info@thirdsectordesign.org.
```

Comment:
thanks for this! can you remove the `[or in the documentation]` bit since there is no documentation for this extension apart from the readme.

Reply:
Done